src/power_grid_model_io/converters/pandapower_converter.py

```diff
@@ -34,6 +36,17 @@
 
 logger = structlog.get_logger(__file__)
 
+pp_curr_version = Version(version("pandapower"))
+pp_ref_version = Version("3.1.2")
+
+
+def get_loss_params_3ph():
+    if pp_curr_version <= pp_ref_version:
+        loss_params = ["p_a_l_mw", "q_a_l_mvar", "p_b_l_mw", "q_b_l_mvar", "p_c_l_mw", "q_c_l_mvar"]
+    else:
+        loss_params = ["pl_a_mw", "ql_a_mvar", "pl_b_mw", "ql_b_mvar", "pl_c_mw", "ql_c_mvar"]
+    return loss_params
```

Comment:
i noticed a problem: pandapower is a test depenency only. Therefore, the converter may be run entirely independently of the pandapower (a use case for that could be if the converter and runner run in different containers). Because of that, it is not known at runtime which pandapower version is used.

Would there be a problem if we always populate both fields?


```suggestion
pp_curr_version = Version(version("pandapower"))
pp_ref_version = Version("3.1.2")


def get_loss_params_3ph():
    return ["p_a_l_mw", "q_a_l_mvar", "p_b_l_mw", "q_b_l_mvar", "p_c_l_mw", "q_c_l_mvar", "pl_a_mw", "ql_a_mvar", "pl_b_mw", "ql_b_mvar", "pl_c_mw", "ql_c_mvar"]
```